scripts/uboot_helper

```diff
@@ -39,6 +39,12 @@ devices = \
         'config': 'pine64_plus_defconfig'
       },
     },
+    'H2+': {
+      'bananapi-m2-zero': {
+        'dtb': 'sun8i-h2-plus-bananapi-m2-zero.dtb',
+        'config': 'bananapi_m2_zero_defconfig'
+      },
+    },
```

Comment:
Split this change in second commit, with subject prefix "uboot_helper:"

Reply:
Done